pages/program-details/[pid].tsx

```diff
@@ -1,133 +1,99 @@
 import { useRouter } from "next/router";
 import React from "react";
-import {
-    Heading,
-    Flex,
-    Badge,
-    Spacer,
-    Text,
-    Button,
-    Spinner,
-    Center,
-} from "@chakra-ui/react";
-import { ClassList } from "src/components/ClassList";
-import Wrapper from "@components/SDCWrapper";
+import { Text } from "@chakra-ui/react";
+import type { ClassCardInfo } from "models/Class";
+import { weekday } from "@prisma/client";
 import { useSession } from "next-auth/client";
-import { BackButton } from "@components/BackButton";
-import useSWR from "swr";
-import CardInfoUtil from "utils/cardInfoUtil";
+import { ProgramInfo } from "@components/ProgramInfo";
 
 export const ProgramDetails: React.FC = () => {
     const [session, loading] = useSession();
     const router = useRouter();
     const { pid } = router.query;
-
-    const fetchWithId = (url, id) =>
```

Comment:
We should keep the SWR/fetch stuff, and only replace the return statement? 
Any suggestions on how to merge this file, Git merge algorithm isnt very helpful :(